devicon.json

```diff
@@ -1,4651 +1,2716 @@
 [
-    {
-      "name": "adonisjs",
-      "tags": [
-          "nodejs",
-          "framework"
+  {
```

Comment:
Please revert all of this changes and only add the changes that add your icon here.

Reply:
@Panquesito7   git reset and rebase was done to update the file.